run/README.namelist

```diff
@@ -51,9 +51,12 @@ Description of namelist variables
  cycling                             = F,       ! whether this run is a cycling run, if so, initializes look-up table for Thompson schemes only
  restart_interval		     = 1440,	! restart output file interval in minutes
  reset_simulation_start              = F,       ! whether to overwrite simulation_start_date with forecast start time
- io_form_history                     = 2,       ! 2 = netCDF 
- io_form_restart                     = 2,       ! 2 = netCDF 
- io_form_input                       = 2,       ! 2 = netCDF
+ io_form_history                     = 2,       ! netCDF format 
+                                     = 11,      ! pnetCDF format
+ io_form_restart                     = 2,       ! netCDF format
+                                     = 11,      ! pnetCDF format
+ io_form_input                       = 2,       ! netCDF format
+                                     = 11,      ! pnetCDF format
```

Comment:
@HathewayWill Isn't the actual file saved by using pnetCDF the same as netCDF, especially from a user's point of view?

Reply:
Good afternoon,

Not 100% certain I understand the question but I'll take a guess at it. 

So the wrf output file naming structure remains the same. However, the namelist.input file has to be designated pnetcdf format for the netcdf files to be written using pnetcdf. 

I asked for clarification on the wrf forum and kwener gave me these suggestions.

https://forum.mmm.ucar.edu/threads/how-to-use-pnetcdf-in-wrf-output-files.12643/#post-30373

As for the output file data since it is using pnetcdf to split the job into a final netcdf file post processing software like ncview and wrf Python can still be used. 


If I misunderstood the question please let me know and I'll try to answer it better.

Reply:
@HathewayWill If pnetCDF file format is the same as netCDF file, albeit they written by different libraries, then I think using netCDF to describe its format is sufficient. Do you?

Reply:
@weiwangncar 

From an article I read online explaining the differences

> Both pnetCDF and netCDF are file formats used for storing and accessing scientific data. However, there are some key differences between the two formats:
> 
> Parallelism: pnetCDF is designed specifically for parallel I/O on distributed-memory systems, while netCDF is primarily intended for single-process I/O. This means that pnetCDF is optimized for high-performance computing and is better suited for applications that require parallel I/O.
> 
> Compression: pnetCDF uses a more efficient compression algorithm than netCDF, which allows it to store data more compactly. This can be an important consideration for applications that generate large amounts of data.
> 
> Language support: While netCDF has support for multiple programming languages, including C, C++, Fortran, Java, and Python, pnetCDF is primarily used with C and Fortran.
> 
> API: The two formats have different APIs for reading and writing data. While the APIs are similar in many ways, there are some important differences that can affect performance and ease of use.
> 
> In summary, pnetCDF is optimized for parallel I/O and has better compression, while netCDF has wider language support and a more general-purpose API. The choice between the two formats will depend on the specific needs of the application.

I think they are different.  Sorry I'm not really the expert on this.  I'll ask @kkeene44 for help.

Reply:
These are the library files I installed for pnetcdf. 

https://github.com/Parallel-NetCDF/PnetCDF

Those are the ones you are mentioning correct?